src/global.css

```diff
@@ -1,10 +1,50 @@
 :root {
   --color-primary: 0deg 0% 0%;
   --color-secondary: 0deg 0% 24%;
-
   --color-hover: 0deg 0% 97%;
+  --color-hover-dark: 0deg 0% 30%;
+  --text-white-color: rgb(255, 255, 255);
+  --background-color-dark: rgb(0, 0, 0);
+  --tw-bg-opacity-dark: 0.4;
 }
 
 html {
   scrollbar-gutter: stable;
 }
+
+.lightThemeIcon {
+  fill: var(--background-color-dark);
+}
+
+.darkThemeIcon {
+  fill: transparent;
+}
```

Comment:
The styles for Icon could live in the `ThemeIcon.astro` file

Reply:
Ok under the style section right ??

Reply:
Yes!